src/config.c

```diff
@@ -409,7 +413,11 @@ int config_print(struct re_printf *pf, const struct config *cfg)
 			 "# SIP\n"
 			 "sip_listen\t\t%s\n"
 			 "sip_certificate\t%s\n"
+#if defined (DEFAULT_CAFILE) || defined (DARWIN) || defined (LINUX)
 			 "sip_cafile\t\t%s\n"
+#else
+			 "#sip_cafile\t\t%s\n"
+#endif
```

Comment:
config_print() should not print config lines starting with '#'. It should simply print the current config.

Reply:
thanks for the hint, removed